src/BuildScriptGenerator/DotNetCore/ProjectFileProviders/ProjectFileHelpers.cs

```diff
@@ -65,6 +70,27 @@ public static bool IsAzureFunctionsProject(XDocument projectFileDoc)
             return HasPackageReference(projectFileDoc, DotNetCoreConstants.AzureFunctionsPackageReference);
         }
 
+        public static bool IsAzureBlazorWebAssemblyProject(XDocument projectFileDoc)
```

Comment:
can you add tests for this

Reply:
sure .. I added a basic buildimage test .. but i will add a test for this as well